pkg/quarterdeck/db/models/users.go

```diff
@@ -434,6 +434,14 @@ func (u *User) loadOrganization(tx *sql.Tx, orgID ulid.ULID) (err error) {
 	if _, ok := u.orgRoles[orgID]; !ok {
 		return ErrUserOrganization
 	}
+
+	// filter u.orgRoles to only the specified orgID
```

Comment:
added filter to only return the roles corresponding to the orgID that was passed or the default organization